[2025-04-04T09:05:03.442+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: download_and_store_images.download_images scheduled__2023-09-24T00:00:00+00:00 [queued]>
[2025-04-04T09:05:03.475+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: download_and_store_images.download_images scheduled__2023-09-24T00:00:00+00:00 [queued]>
[2025-04-04T09:05:03.476+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-04-04T09:05:03.477+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2025-04-04T09:05:03.478+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-04-04T09:05:03.548+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): download_images> on 2023-09-24 00:00:00+00:00
[2025-04-04T09:05:03.559+0000] {standard_task_runner.py:55} INFO - Started process 2186 to run task
[2025-04-04T09:05:03.567+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'download_and_store_images', 'download_images', 'scheduled__2023-09-24T00:00:00+00:00', '--job-id', '536', '--raw', '--subdir', 'DAGS_FOLDER/dag_data_collect.py', '--cfg-path', '/tmp/tmppggknx1h']
[2025-04-04T09:05:03.568+0000] {standard_task_runner.py:83} INFO - Job 536: Subtask download_images
[2025-04-04T09:05:03.611+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-04-04T09:05:03.689+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2025-04-04T09:05:03.838+0000] {task_command.py:388} INFO - Running <TaskInstance: download_and_store_images.download_images scheduled__2023-09-24T00:00:00+00:00 [running]> on host 91a2461e5846
[2025-04-04T09:05:04.121+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=download_and_store_images
AIRFLOW_CTX_TASK_ID=download_images
AIRFLOW_CTX_EXECUTION_DATE=2023-09-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-24T00:00:00+00:00
[2025-04-04T09:05:04.388+0000] {python.py:177} INFO - Done. Returned value was: [None, None]
[2025-04-04T09:05:04.448+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=download_and_store_images, task_id=download_images, execution_date=20230924T000000, start_date=20250404T090503, end_date=20250404T090504
[2025-04-04T09:05:04.514+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-04-04T09:05:04.894+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
